popup.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+  <head>
+    <title>The Ed Extension</title>
+    <style>
+      body {
+        width: 350px;
+        font-family: "Arial", sans-serif;
+      }
+    </style>
+  </head>
+  <body style="background-color: rgba(128,77,160,255)">
+    <h2 style="color: white;">The Ed Extension</h2>
+    <p style="color: white; font-family: 'Courier New', monospace;white-space: pre;">
+_____    _    ___  _           _    <br>
+|  ___|  | |  / _ \| |         | |  <br>
+| |__  __| | / /_\ \ | ___ _ __| |_ <br>
+|  __|/ _` | |  _  | |/ _ \ '__| __|<br>
+| |__| (_| | | | | | |  __/ |  | |_ <br>
+\____/\__,_| \_| |_/_|\___|_|   \__|<br>
+                                    <br>
+Developed by Jacob Kerr<br>
+https://github.com/jakejack13/edalert/tree/main<br>
```

Comment:
I would change this to the repo link itself instead of the main branch tree
[https://github.com/jakejack13/edalert](https://github.com/jakejack13/edalert)